optuna/visualization/_optimization_history_matplotlib.py

```diff
@@ -0,0 +1,65 @@
+from matplotlib.figure import Figure
+from matplotlib import pyplot as plt
+
+from optuna.logging import get_logger
+from optuna.study import Study
+from optuna.study import StudyDirection
+from optuna.trial import TrialState
+
+_logger = get_logger(__name__)
+
+
+def _get_optimization_history_plot_matplotlib(study: Study) -> Figure:
+    """Plot optimization history of all trials in a study with matplotlib.
+
+    Args:
+        study:
+            A :class:`~optuna.study.Study` object whose trials are plotted for their objective
+            values.
+
+    Returns:
+        A :class:`matplotlib.figure.Figure` object.
+    """
+
+    # Setup
```

Comment:
A nit, but could you use sentences instead of a word in the code comment?
The following is just an example:
```suggestion
    # Setup the graph style.
```